services/html-scraper.service.ts

```diff
@@ -0,0 +1,34 @@
+import Axios from "axios";
+import { config } from "../config";
+
+export class HtmlScraperService {
+    static async scrape(url: string): Promise<string> {
```

Comment:
I extracted logic for scraping into this service. Its sole purpose is for given URL return the scraped data.

Reply:
I suggest injecting config through ctor, otherwise this in untestable (unmockable)